jobs/webcompat-kb/webcompat_kb/main.py

```diff
@@ -609,7 +692,7 @@ def update_kb_ids(self, ids):
         table = self.client.get_table(kb_bugs_table)
         logging.info(f"Loaded {table.num_rows} rows into {table}")
 
-    def update_relations(self, relations):
+    def update_relations(self, relations, relation_config):
```

Comment:
I don't remember why I didn't add type annotations here, but if we could it would be nice :)

Reply:
Added:)